senders/discord/send.go

```diff
@@ -0,0 +1,151 @@
+package discord
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/moira-alert/moira"
+)
+
+const (
+	messageMaxCharacters = 2000
+)
+
+var (
+	mdHeaderRegex = regexp.MustCompile(`(?m)^\s*#{1,}\s*(?P<headertext>[^#\n]+)$`)
+)
+
+// SendEvents implements pushover build and send message functionality
+func (sender *Sender) SendEvents(events moira.NotificationEvents, contact moira.ContactData, trigger moira.TriggerData, plot []byte, throttled bool) error {
+	data := &discordgo.MessageSend{}
+	data.Content = sender.buildMessage(events, trigger, throttled)
+	if len(plot) > 0 {
+		data.File = sender.buildPlot(plot)
+		data.Embed = &discordgo.MessageEmbed{
+			Image: &discordgo.MessageEmbedImage{
+				URL: "attachment://Plot.jpg",
+			},
+		}
+	}
+	sender.logger.Debugf("Calling discord with message %s", data.Content)
+	_, err := sender.session.ChannelMessageSendComplex(contact.Value, data)
```

Comment:
- How does channelID differ from channelName? And usage of which parameter could be more user-friendly?
- How will we be sending messages to users in private?

Reply:
[This](https://support.discordapp.com/hc/en-us/articles/206346498-Where-can-I-find-my-User-Server-Message-ID-) is the official article from discord support on how to get a channel ID. It also mentions how to get the channel ID of a private chat with an individual user. Using channel name is more user-friendly but it's not advisable as mentioned [here](https://stackoverflow.com/questions/41515134/discord-bot-cant-get-channel-by-name) and the discord API docs also don't mention anything about getting a channel by its name or getting a list of all channels and searching for it.

Reply:
So what's the arguments you see to use channel id instead of channel name?

Reply:
I am not saying that this is only right way even though i've already implemented such channel-name-to-id resolver (in my local repo)
I'm just interested in advantages and disadvantages of both methods

Reply:
1. There can be multiple channels of the same name. 
2. There is no straight forward way (as far as I could find) of resolving a channel name to its unique ID. Can you please share the method you used?
The advantage of using channel names is obviously user-friendliness.

Reply:
> Can you please share the method you used?
Of course, No!

Reply:
> Of course, No!

I don't understand!? Why? By local repo did you mean closed source(I couldn't find anything in your fork)? Or do you want me to figure it out myself? :)
But the fact that multiple channels with the same name can exist should by default rule out the usage of channel names. Shouldn't it?
![image](https://user-images.githubusercontent.com/31558262/60344987-934ea780-99d5-11e9-937c-215b1c1e837a.png)

Reply:
> Or do you want me to figure it out myself?

Of course, Yes!

> multiple channels with the same name can exist

Take a look on our Telegram sender, it has the same entity called Group chats
You can see how do we store chat id's

Please take more time on this feature. 
Try to figure out how we can make life of our users easier in this particular case